modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -627,7 +678,12 @@ export default {
       ) {
         this.currentPage++;
         this.isScrollLoading = true;
-        const result = await this.getMedia({ lock });
+        const result = await this.getMedia({
+          lock,
+          // Tell the load handler to request more items if it finds
+          // duplicates in the response, only after an upload.
+          loadMoreIfSkipped: this.uploaded
```

Comment:
I don't understand, this option is never used in the method `getMedia`.
edit: Ok I just understood that `getMedia` returns the options in the result. It's a bit hard to track.